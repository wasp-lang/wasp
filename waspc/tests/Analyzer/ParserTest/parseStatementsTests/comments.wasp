  // This is some // comment
/* comment
  span//ning
multi/*ple lines */
test /* *hi* */ Decl 42  // One more comment
// And here is final comment
