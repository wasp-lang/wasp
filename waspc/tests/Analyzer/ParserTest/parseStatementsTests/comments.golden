(AST
  (Decl@5:1-23 type=test name=Decl
    (Integer@5:22-23 value=42)
  )
)
